test/blackbox-tests/test-cases/pkg/broken-symlink-in-dependency.t

```diff
@@ -0,0 +1,75 @@
+Test that dune can handle the case where a dependency's source contains a
+symlink with a missing destination.
+
+  $ . ./helpers.sh
+
+Define a package foo containing a broken symlink.
+  $ mkdir foo
+  $ touch foo/a.txt
+  $ ln -s non_existent foo/b.txt
+
+Define a package foo containing a broken symlink.
+  $ mkdir bar
+  $ touch bar/a.txt
+  $ ln -s non_existent bar/b.txt
+  $ tar pczf bar.tar.gz bar
```

Comment:
I had to look up `p` and according to the man page this is `-p, --preserve-permissions, --same-permissions`. Is this required for the test? I'd probably just create a tarball without compression (to save CPU cycles compressing and decompressing).

Reply:
You're right, somehow `p` made its way into my muscle memory for creating a tarball, but it is unnecessary here.